keystone-init/keystone_init.py

```diff
@@ -554,6 +592,23 @@ def grant_group_role(client, role_id, group, project):
     client.roles.grant(role_id, group=group, project=project)
 
 
+@retry()
+def grant_group_domain_role(client, role_id, group, domain):
+    """Grant a role to a group on a project
```

Comment:
Should probably be domain here, not project.

Reply:
good catch, fixed